deploy/docker/deploy-internalstage.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# NOTE that ic-docker-registry.nrel.gov:5000/ registry server is available only w/in nrel.  also, ssl certs for internal domains are a major hassle.
+# allow your local docker to access ic-docker-registry w/out cert:
+# 1. open docker preferences
+# 2. Daemons: add 'ic-docker-registry:5000' as an insecure registry
+
+#build images
+export BT_DOCKER_REGISTRY_URI=ic-docker-registry.nrel.gov:5000/
+export BT_ENV=stage
+docker-compose -f ./deploy/docker/docker-compose-staging.yml build
```

Comment:
docker-compose-staging doesn't exist yet does it? I do not see it as part of this PR.